.github/workflows/merge-check.yml

```
@@ -0,0 +1,33 @@
+name: Merge checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    name: Build project
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.head_commit.message, 'ci skip') }}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Cache Gradle Caches
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches/
+          key: cache-gradle-cache
+
+      - name: Cache Gradle Wrapper
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/wrapper/
+          key: cache-gradle-wrapper
```

```suggestion
      - name: Cache Gradle Files
        uses: actions/cache@v2
        with:
          path: |
            ~/.gradle/caches/
            ~/.gradle/wrapper/
          key: cache-gradle-files
```

Cache action is able to handle multiple paths. See https://github.com/cortinico/kotlin-android-template/pull/29/

---

Sounds good to me. Do you want to make the same change in `dex-test-parser` so they're consistent? Since I copied your PR from there :wink: